api/services/AuthenticationService.cs

```diff
@@ -7,21 +8,20 @@
 
 namespace LiteralLifeChurch.LiveStreamingApi.services
 {
-    class AuthenticationService
+    public class AuthenticationService
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5532272)